konig-dao-sql-runtime/src/main/java/io/konig/sql/runtime/BigQueryShapeReadService.java

```diff
@@ -43,14 +42,15 @@
 import io.konig.dao.core.DaoException;
 import io.konig.dao.core.Format;
 
-public class BigQueryShapeReadService extends SqlShapeReadService {
-	private static final SimpleDateFormat dateFormatUTC = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'");
-
+public class BigQueryShapeReadService extends SqlShapeReadService {
 	private BigQuery bigQuery;
+	private DateTimeZone timeZone;
 	
 	public BigQueryShapeReadService(EntityStructureService structureService, BigQuery bigQuery) {
 		super(structureService);
 		this.bigQuery = bigQuery;
+		this.timeZone = DateTimeZone.UTC;
```

Comment:
What's the point of storing timeZone as a field if you are just going to set it equal to a static value?  Why not just use the  static value directly?